privaterelay/views.py

```diff
@@ -55,7 +56,9 @@
 def home(request):
     if (request.user and not request.user.is_anonymous):
         return redirect(reverse('profile'))
-    return render(request, 'home.html')
+    return render(request, 'home.html', {
+        'ftl_bundle': main_bundle,
+    })
```

Comment:
This is some copy-pasta left over from the docs when I first installed it. I think this can be removed (as the other pages don't need them to work). I'll confirm and revise as necessary.

Reply:
Removed!